cpp/src/gandiva/precompiled/string_ops.cc

```diff
@@ -2837,21 +2840,30 @@ const char* soundex_utf8(gdv_int64 ctx, const char* in, gdv_int32 in_len,
   for (int i = start_idx, l = in_len; i < l; i++) {
```

Comment:
What is need for the temporary variable l

Reply:
don't need, is from old version from Augusto.
I removed